Source/Core/Core/PowerPC/JitArm32/JitArm_FloatingPoint.cpp

```diff
@@ -27,7 +27,7 @@ void JitArm::fctiwx(UGeckoInstruction inst)
 {
 	INSTRUCTION_START
 	JITDISABLE(bJITFloatingPointOff);
-	FALLBACK_IF(true);
+	
```

Comment:
So, fctiwx is working, but fctiwzx isn't?  What testing have you done to prove this?  On a similar note, it would be much more useful if you could say that fctiwzx doesn't work for the float value 0xXXXXXXXX rather than saying that it breaks Animal Crossing.

We have a framework for running tests like this; see https://github.com/dolphin-emu/hwtests ; if you need help figuring out how to use it, hop on to #dolphin-dev .

(We have very high standards for JIT changes, especially from new developers, because there's a history of people implementing instructions in the JIT and messing up in subtle ways.)